wraith/configs/capture.yaml

```diff
@@ -0,0 +1,78 @@
+##############################################################
+##############################################################
+# This is an example configuration provided by Wraith.
+# Feel free to amend for your own requirements.
+# ---
+# This particular config is intended to demonstrate how
+# to use Wraith in 'capture' mode, which is best suited to
+# comparing a test and live version of the same website.
+#
+# `wraith capture capture.yaml`
+#
+##############################################################
+##############################################################
+
+# (required) The engine to run Wraith with. Examples: 'phantomjs', 'casperjs', 'slimerjs'
+browser: "phantomjs"
+
+# (required) The domains to take screenshots of.
+domains:
+  local:       "http://localhost:5000"
+  live:        "https://www.pensionwise.gov.uk"
+
+# # (required) The paths to capture. All paths should exist for both of the domains specified above.
+paths:
+  home:     /
+  styleguide:    /styleguide
```

Comment:
Hey I just met you, and this is crazy, but here's some whitespace, delete it maybe?